src/main/docker/scripts/wait-for.sh

```diff
@@ -0,0 +1,191 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/bin/sh

# Taken from https://github.com/eficode/wait-for
```
A comment here would be helpful to know in the future where to look for updates in case something breaks.

Reply:
Good point. I would maybe even create a wait-for.md in the same directory and write why we use it and where it's from and a version and then replacing the script should be easy if it will ever be done :smirk:

Reply:
@b-fein added a short description with source and version